### ISSUE TITLE
v6 handleSubmit triggers Object.values and isn't converted into ES5

### ISSUE THREAD

**Issue by darylhedley** [erikras/redux-form#907]
Having a problem with the v6 alpha version of Redux Form. Chrome doesn't like Object.values and maybe this is an issue with Webpack/Babel in my application. Should this be converted into an ES5 method or use lodash _.values?

**Comment by babsonmatt** [comment]
Do you need `require('babel-polyfill');` in your webpack entry point?

**Comment by darylhedley** [comment]
Not too sure. I currently have:

**.babelrc**
```
{
  "presets": ["react", "es2015", "stage-1"]
}
```

**webpack.config.js**
```
module: {
        loaders: [{
            exclude: /node_modules/,
            loader: 'babel'
        }]
    },
```

**Comment by erikras** [comment]
This is has been fixed, and will be released in an upcoming alpha release.

**Comment by darylhedley** [comment]
Thanks! Also just to say we've been using the Alpha release and it's amazing - the new ``Field`` component is great.

**Comment by erikras** [comment]
Fix published as [`v6.0.0-alpha-7`](https://github.com/erikras/redux-form/releases/tag/v6.0.0-alpha.7).

**Comment by darylhedley** [comment]
Ace! Thanks!
